### PR TITLE
v2/acl: add `Service` filter type

### PR DESCRIPTION
**Pull request by fyrchik**
Depends on nspcc-dev/neofs-api#173

Signed-off-by: Evgenii Stratonikov <evgeniy@nspcc.ru>

### CODE REVIEW

File: v2/acl/grpc/types.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.23.0
-// 	protoc        v3.14.0
```
Comment (alexvanin) [inline v2/acl/grpc/types.pb.go:4]: I suppose it's better to recompile all protobuf files then, or use protoc v3.14. 

/cc @cthulhu-rider @carpawell

**Review by alexvanin — COMMENTED** [review]
Support it in `pkg/acl/eacl/enums.go` as well.

File: v2/acl/grpc/types.pb.go (hunk 1 above)
Comment (carpawell) [reply v2/acl/grpc/types.pb.go:4]: i'd rather keep it `v3.14.0` and recompile all files in a separate PR

Comment (fyrchik) [reply v2/acl/grpc/types.pb.go:4]: What's the problem with leaving this as it is? protoc-gen-go is specified in our makefile, protoc is not, so I used the system default.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**